src/main/java/frc/robot/RobotContainer.java

```diff
@@ -63,6 +64,10 @@ private void configureBindings() {
   }
 
   public Command getAutonomousCommand() {
+    if(Robot.isSimulation()) {
+      //drivetrainSubsystem.resetPose(new Pose2d(new Translation2d(4, -7), Rotation2d.fromRadians(0.0)));
+      return new AutoGenerator(drivetrainSubsystem).getExitAndBalance(StartLocation.ONE);
+    }
```

Comment:
Is this needed or could it still be based on our user selection from the Shuffleboard